content/api/introduction/pagination/page-arguments.mdx

```diff
@@ -57,6 +57,17 @@ server
     console.error(err);
   });
 ```
+  
+```python
+from stellar_sdk import Server
+server = Server("https://horizon.stellar.org")
+
+try:
+    print(server.operations().for_ledger("26478723").cursor("113725249324879872").limit(5).order("asc").call())
```

Comment:
I would prefer not going over the line length, something like
```suggestion
    print(server.operations()
        .for_ledger("26478723")
        .cursor("113725249324879872")
        .limit(5)
        .order("asc")
        .call())
```
would look cleaner on the page.